scenarios/kubernetes_e2e_test.py

```diff
@@ -157,86 +156,85 @@ def test_local(self):
+        args = kubernetes_e2e.parse_args(['--mode=docker', '--check-leaked-resources=false'])
         with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
             kubernetes_e2e.main(args)
             self.assertIn('--check-leaked-resources=false', self.callstack[-1])
             self.assertIn('-e FAIL_ON_GCP_RESOURCE_LEAK=false', self.callstack[-1])
 
     def test_check_leaks(self):
         """Ensure --check-leaked-resources=true sends flag to kubetest."""
-        args = self.parser.parse_args(['--check-leaked-resources=true', '--mode=local'])
+        args = kubernetes_e2e.parse_args(['--check-leaked-resources=true', '--mode=local'])
         with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
             kubernetes_e2e.main(args)
             self.assertIn('--check-leaked-resources=true', self.callstack[-1])
             self.assertEquals('false', self.envs.get('FAIL_ON_GCP_RESOURCE_LEAK'))
 
     def test_check_leaks_false(self):
         """Ensure --check-leaked-resources=true sends flag to kubetest."""
-        args = self.parser.parse_args(['--check-leaked-resources=false', '--mode=local'])
+        args = kubernetes_e2e.parse_args(['--check-leaked-resources=false', '--mode=local'])
         with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
             kubernetes_e2e.main(args)
             self.assertIn('--check-leaked-resources=false', self.callstack[-1])
             self.assertEquals('false', self.envs.get('FAIL_ON_GCP_RESOURCE_LEAK'))
 
     def test_check_leaks_default(self):
         """Ensure --check-leaked-resources=true sends flag to kubetest."""
-        args = self.parser.parse_args(['--check-leaked-resources', '--mode=local'])
+        args = kubernetes_e2e.parse_args(['--check-leaked-resources', '--mode=local'])
         with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
             kubernetes_e2e.main(args)
-            self.assertIn('--check-leaked-resources=true', self.callstack[-1])
+            self.assertIn('--check-leaked-resources', self.callstack[-1])
             self.assertEquals('false', self.envs.get('FAIL_ON_GCP_RESOURCE_LEAK'))
 
     def test_check_leaks_unset(self):
         """Ensure --check-leaked-resources=true sends flag to kubetest."""
-        args = self.parser.parse_args(['--mode=local'])
+        args = kubernetes_e2e.parse_args(['--mode=local'])
         with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
             kubernetes_e2e.main(args)
-            self.assertIn('--check-leaked-resources=false', self.callstack[-1])
+            self.assertNotIn('--check-leaked-resources', self.callstack[-1])
             self.assertEquals('false', self.envs.get('FAIL_ON_GCP_RESOURCE_LEAK'))
 
-    def test_extract_multiple(self):
-        """Ensure multiple --extract flags are accepted."""
-        args = self.parser.parse_args(['--extract=a', '--extract=b'])
```

Comment:
Yeah, I was more worried about the human interaction case (and not just what we have in config files). I personally use spaces instead of = on the command-line all the time, and if that changed the behavior in subtle ways, that would be infuriating. :-)